src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,122 @@
+      <tbody>
+        {visibleTodos.map(({
+          id,
+          title,
+          completed,
+          userId,
+        }) => (
+          <tr
+            key={id}
+            data-cy="todo"
+            className={classNames({
+              'has-background-info-light': isSelected(id),
+            })}
+          >
+            <td className="is-vcentered">{id}</td>
             <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">quo adipisci enim quam ut ab</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">9</td>
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">molestiae perspiciatis ipsa</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
+              {completed && (
+                <span className="icon" data-cy="iconCompleted">
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">10</td>
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
+              )}
             </td>
 
             <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
+              <p className={classNames({
+                'has-text-danger': !completed,
+                'has-text-success': completed,
+              })}
+              >
+                {title}
+              </p>
             </td>
 
             <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
+              <button
+                data-cy="selectButton"
+                className="button"
+                type="button"
+                onClick={() => dispatch(
+                  currentTodoActions.setTodo({
+                    id,
+                    title,
+                    completed,
+                    userId,
+                  }),
+                )}
```

Comment:
Anonimous function in JSX.
Better to make function for it and pass just reference to onClick prop